run.go

```diff
@@ -69,8 +70,13 @@ const (
 	CollectorJSON     = "json"
 	CollectorInfluxDB = "influxdb"
 	CollectorCloud    = "cloud"
+
+	SummaryFormatText = SummaryFormat(iota)
```

Comment:
Consistency: else where we use the form `SummaryFormatText SummaryFormat = iota`.